models/DataObject/Classificationstore.php

```diff
@@ -198,14 +198,18 @@ public function setLocalizedKeyValue($groupId, $keyId, $value, $language = null)
 
         if (!$this->isFieldDirty('_self')) {
             if ($this->object) {
-                $oldData = $this->items[$groupId][$keyId][$language];
```

Comment:
Would say simply adding a trailing ?? null
would be sufficient.

Reply:
Done!